cmd/oras/root/cp_test.go

```diff
@@ -132,7 +132,7 @@ func Test_doCopy(t *testing.T) {
 	opts.From.Reference = memDesc.Digest.String()
 	dst := memory.New()
 	builder := &strings.Builder{}
-	printer := output.NewPrinter(builder)
+	printer := output.NewPrinter(builder, false)
```

Comment:
Although it's not used, it's better to set the value to the command option.
```suggestion
	printer := output.NewPrinter(builder, opts.Verbose)
```

Reply:
Yes, done.